relay-server/src/utils/rate_limits.rs

```diff
@@ -237,15 +244,17 @@ impl Default for CategoryLimit {
 #[derive(Default, Debug)]
 pub struct Enforcement {
     /// The event item rate limit.
-    event: CategoryLimit,
+    pub event: CategoryLimit,
```

Comment:
Across modules, we should build all APIs as public APIs - mostly to maintain high code quality.

Reply:
Did this in https://github.com/getsentry/relay/pull/1517/commits/817c70355665ad93cd2bfd373f192835a349b946 but honestly, I don't think this is better from a code quality point of view:

- This is now no longer symmetric: there is only one field with an accessor function.  This is because the linter does not like unused methods (good!).  So now the user has to guess why only one of them is made available.  Weird.
- The caller now has `field.accessor().method()`, this is entirely subjective, but `field.field.method()` is faster to parse for a human because really what's written is `field.method().method()` and you have to mentally translate this to `field.accessor().method()` to understand what's going on.

But I don't mind, happy to leave it in.